# This service is meant to label an image using user clicks.

#Call

#Number of requested points.
int32 requestedPoints

# image that needs to be labeled.
sensor_msgs/Image imageQuery

---

#list of response points:
geometry_msgs/Polygon pointsResp


#labeled image response
sensor_msgs/Image imageResp
